astropy/table/tests/test_table.py

```diff
@@ -427,6 +427,39 @@ def test_10(self, table_types):
 
 
 @pytest.mark.usefixtures('table_types')
+class TestAddName(SetupData):
+
+    def test_1(self, table_types):
```

Comment:
Could you make the names of the test functions a bit more descriptive? E.g., here,
```
def test_override_name(...)
```